corehq/apps/app_manager/app_translations/app_translations.py

```diff
@@ -692,8 +692,8 @@ def has_translation(row_, langs):
         for label in skip_label:
             msgs.append((
                 messages.error,
-                "You must provide at least one translation" +
-                " for the label '%s' in sheet '%s'" % (label, sheet.worksheet.title)
+                _("You must provide at least one translation"
+                  " for the label '%s' in sheet '%s'") % (label, sheet.worksheet.title)
```

Comment:
FYI, I saw this when I updated translations today:

```
./corehq/apps/app_manager/app_translations/app_translations.py:693: warning: 'msgid' format string with unnamed arguments cannot be properly localized:
                                                                             The translator cannot reorder the arguments.
                                                                             Please consider using a format string with named arguments,
                                                                             and a mapping instead of a tuple for the arguments.
```